tests/validators.py

```diff
@@ -535,6 +535,12 @@ def validate_openfaas():
     wait_for_pod_state("", "openfaas", "running", label="app=gateway")
 
 
+def validate_argocd():
+    """
+    Validate argocd
+    """
+    wait_for_pod_state("", "argocd", "running", label="app.kubernetes.io/component=server")
+
```

Comment:
Final thing, add an extra new line to make `black` happy

Reply:
done 😄